interfaces/Config/templates/config_notify.tmpl

```diff
@@ -142,7 +142,7 @@
         <div class="col2">
             <h3>$T('section-NC')</h3>
         </div><!-- /col2 -->
-        <div class="col1" <!--#if $have_ncenter then '' else 'style="display:none;"'#-->">
+        <div class="col1" <!--#if $have_ncenter then '' else 'style="display:none;'#-->">
```

Comment:
This change is still not right;  in case of no ncenter, the end result will be:
<div class="col1" ">
Instead, the " at the end of the line should be removed.